RadarSDK/Include/Radar.h

```diff
@@ -765,6 +774,34 @@ typedef void (^_Nonnull RadarRouteMatrixCompletionHandler)(RadarStatus status, R
                        units:(RadarRouteUnits)units
            completionHandler:(RadarRouteMatrixCompletionHandler)completionHandler NS_SWIFT_NAME(getMatrix(origins:destinations:mode:units:completionHandler:));
 
+/**
+ Sends a custom event.
+
+ @param name The name of the event.
+ @param metadata The metadata associated with the event.
+ @param completionHandler A completion handler.
+
+ @see https://radar.com/documentation/api#send-event
+ */
++ (void)sendEvent:(NSString *)name
+     withMetadata:(NSDictionary *_Nullable)metadata
```

Comment:
There are future iterations we want to improve on but this requires schema changes / backfills for our events table in athena. Specifically the events table does not have a column for metadata or a user's external ID
```
CREATE EXTERNAL TABLE `events_by_project_v2`(
  `_id` string COMMENT '', 
  `createdat` timestamp COMMENT '', 
  `actualcreatedat` timestamp COMMENT '', 
  `organization` string COMMENT '', 
  `live` boolean COMMENT '', 
  `locationaccuracy` double COMMENT '', 
  `confidence` int COMMENT '', 
  `place` string COMMENT '', 
  `placecategories` string COMMENT '', 
  `placechainname` string COMMENT '', 
  `placechainslug` string COMMENT '', 
  `placegroup` string COMMENT '', 
  `placename` string COMMENT '', 
  `alternateplaces` array<string> COMMENT '', 
  `placecandidates` array<string> COMMENT '', 
  `duration` double COMMENT '', 
  `longitude` double COMMENT '', 
  `latitude` double COMMENT '', 
  `user` string COMMENT '', 
  `geofence` string COMMENT '', 
  `geofencedescription` string COMMENT '', 
  `geofencetag` string COMMENT '', 
  `geofenceexternalid` string COMMENT '', 
  `region` string COMMENT '', 
  `countryregionname` string COMMENT '', 
  `countryregioncode` string COMMENT '', 
  `countryregion` string COMMENT '', 
  `dmaregion` string COMMENT '', 
  `dmaregionname` string COMMENT '', 
  `dmaregioncode` string COMMENT '', 
  `stateregion` string COMMENT '', 
  `stateregionname` string COMMENT '', 
  `stateregioncode` string COMMENT '', 
  `postalcoderegion` string COMMENT '', 
  `postalcoderegioncode` string COMMENT '', 
  `replayed` boolean COMMENT '', 
  `type` string COMMENT '', 
  `payload` string COMMENT '', 
  `timezoneoffset` int COMMENT '', 
  `trip` string COMMENT '', 
  `beacon` string COMMENT '', 
  `foreground` boolean COMMENT '', 
  `locationaccuracyauthorization` string COMMENT '', 
  `locationauthorization` string COMMENT '', 
  `devicetype` string COMMENT '', 
  `devicemake` string COMMENT '', 
  `devicemodel` string COMMENT '', 
  `deviceos` string COMMENT '', 
  `sdkversion` string COMMENT '', 
  `stopped` boolean COMMENT '', 
  `beacontag` string COMMENT '', 
  `beaconexternalid` string COMMENT '', 
  `beacontype` string COMMENT '', 
  `beaconmajor` string COMMENT '', 
  `beaconminor` string COMMENT '', 
  `beaconuuid` string COMMENT '', 
  `tripexternalid` string COMMENT '', 
  `tripdestinationgeofencetag` string COMMENT '', 
  `tripdestinationgeofenceexternalid` string COMMENT '')
```

Reply:
helpful to have the schema, thanks for providing

yeah and no worries re: backfill, this is a net new capability for custom events, as existing events don't have any `metadata` attached - just needs to be there going forward